src/App.tsx

```diff
@@ -1,24 +1,190 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState, useRef } from 'react';
+import classNames from 'classnames';
+import { deleteTodo, getTodos, postTodo } from './api/todos';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList';
+import { Footer } from './components/Footer';
+import { ErrorsNotification } from './ErrorsNotification';
+import { Filter } from './types/Filter';
+import { Error } from './types/Error';
 
-const USER_ID = 0;
+const USER_ID = 6910;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo []>([]);
+  const [error, setError] = useState('');
+  const [filter, setFilter] = useState(Filter.All);
+  const [title, setTitle] = useState('');
+  const [tempTodo, setTempTodo] = useState<Todo | null> (null);
+  const [todosTransform, setTodosTransform] = useState<number[]>([]);
+  const [isDeleting, setIsDeleting] = useState<boolean>(false);
+
+  const focusInput = useRef<HTMLInputElement | null>(null);
+
+  const errorTimeoutId = () => setTimeout(() => {
```

Comment:
Clear all times on the component unmount
https://bobbyhadz.com/blog/react-cleartimeout